app/models/datawarehouse_node.rb

```diff
@@ -0,0 +1,9 @@
+class DatawarehouseNode < ApplicationRecord
```

Comment:
@durandom I don't see this as vendor specific. Most Datawarehouses will have nodes. Hawkular seems an odd provider because it has started as a middleware manager that handled these things. 
@simon3z Do you have an opinion here?